floor_generator/lib/misc/extension/dart_type_extension.dart

```diff
@@ -0,0 +1,18 @@
+// TODO #375 delete once dependencies have migrated
+// ignore_for_file: import_of_legacy_library_into_null_safe
+import 'package:analyzer/dart/element/nullability_suffix.dart';
+import 'package:analyzer/dart/element/type.dart';
+
+extension DartTypeExtension on DartType {
+  // TODO #375 test
+  bool get isNullable {
+    final nullabilitySuffix = this.nullabilitySuffix;
+    switch (nullabilitySuffix) {
+      case NullabilitySuffix.question:
+      case NullabilitySuffix.star: // support legacy code without non-nullables
```

Comment:
This will cause problems the way `isNullable` is used right now because it won't allow usages from legacy code as we're always throwing.